test/Responses/DeleteResponses.cs

```diff
@@ -0,0 +1,15 @@
+using System.Net;
+using System.Net.Http;
+using System.Text;
+
+namespace Ovh.Test.Responses
+{
+    public static class Delete
+    {
+        public static string nullAsJsonString = "null";
+        public static HttpResponseMessage nullAsHttpResponseMessage = new HttpResponseMessage{
+            Content = new StringContent(nullAsJsonString, Encoding.UTF8, "application/json"),
```

Comment:
Could `"null"` be inlined here? Or at least private?
Could both be readonly?
Could `new HttpResponseMessage { ... }` be in a static method to avoid code duplication?

(Note: may apply to other response classes)

Reply:
- Might be, but I might refer to it from future tests and I'm not sure that it would be a big perf improvement anyway :)
- Yes, done
- Done